media/js/base/stub-attribution.js

```diff
@@ -155,6 +155,38 @@ if (typeof window.Mozilla === 'undefined') {
         $.get(SERVICE_URL, data).done(StubAttribution.onRequestSuccess);
     };
 
+    /**
+     * Returns a browser name based on coarse UA string detection for only major browsers.
+     * Other browsers (or modified UAs) that have strings that look like one of the top default user agent strings are treated as false positives.
+     * @param {String} ua - Optional user agent string to facilitate testing.
+     * @returns {String} - Browser name.
+     */
+    StubAttribution.getUserAgent = function(ua) {
+        ua = typeof ua !== 'undefined' ? ua : navigator.userAgent;
+
+        if (/MSIE|Trident/i.test(ua)) {
+            return 'ie';
+        }
+
+        if (/Edg|Edge/i.test(ua)) {
+            return 'edge';
+        }
+
+        if (/Firefox/.test(ua)) {
+            return 'firefox';
+        }
+
+        if (/Opera/.test(ua)) {
+            return 'opera';
```

Comment:
No need to fix this, as we didn't list Opera in the bug and this is acceptable as a false positive. FYI, here's the user agent from Opera on mac:

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36 OPR/66.0.3515.72

and windows:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.87 Safari/537.36 OPR/66.0.3515.44

Reply:
If Opera isn't a concern I'll go ahead and remove this altogether 👍